ilc/router/Router.js

```diff
@@ -43,7 +45,9 @@ module.exports = class Router {
             throw new errors.NoRouteMatchError();
         }
 
-        return _defaultsDeep({}, this.#specialRoutes[404]);
+        return _defaultsDeep({
+            isSpecialRoute: true,
```

Comment:
Add ability to determine type of special route

Reply:
Updated